extra/1-currency-conversion.js

```diff
@@ -15,7 +17,10 @@ function convertToUSD() {}
   They have also decided that they should add a 1% fee to all foreign transactions, which means you only convert 99% of the £ to BRL.
 */
 
-function convertToBRL() {}
+function convertToBRL(price) {
+  price = price * 0.99 * 5.7;
```

Comment:
when declaring a variable, must use const or let, otherwise it becomes a global variable which can cause a bug.